### PR TITLE
User Defined Ingress Controller updates

### PR DESCRIPTION
**Pull request by ironcladlou**
Update the enhancement to reflect the most recent changes material to
the implementation and plan.

### CODE REVIEW

**Comment by ironcladlou** [comment]
@openshift/sig-network-edge

File: enhancements/user-defined-default-ingress-controller.md [hunk 1]
```diff
@@ -23,9 +23,9 @@ superseded-by:
 ## Release Signoff Checklist
 
 - [x] Enhancement is `implementable`
-- [ ] Design details are appropriately documented from clear requirements
-- [ ] Test plan is defined
-- [ ] Graduation criteria for dev preview, tech preview, GA
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
```
Comment (Miciah) [inline enhancements/user-defined-default-ingress-controller.md]: It's still a TODO though.

Comment (ironcladlou) [reply enhancements/user-defined-default-ingress-controller.md]: Fixed

**Comment by ironcladlou** [comment]
@Miciah fixed up, PTAL

**Pull request closed without merging**